api/concept/create.py

```
@@ -1,34 +1,39 @@
-            SUG.REQ.PST["Kes"],
-            SUG.REQ.PST["Kde"]
-        ]
-    ))
-    return SUG.REQ.RES.get()
\ No newline at end of file
+#
+#   HANDLER
+#
+
+# HANDLER -> LOAD
+from website import *
```

Ruff (0.11.9)

`from website import *` used; unable to detect undefined names

(F403)